docs/index.asciidoc

```diff
@@ -98,6 +98,9 @@ TIP: Set the `target` option to avoid potential schema conflicts.
 
 This plugin supports the following configuration options plus the <<plugins-{type}s-{plugin}-common-options>> and the <<plugins-{type}s-{plugin}-deprecated-options>> described later.
```

Comment:
```suggestion
This plugin supports these configuration options plus the <<plugins-{type}s-{plugin}-common-options>>  described later.
```

Reply:
This suggestion:
* Removes antiquated language ("the following..." )
* Removes inaccurate info suggesting that we still support deprecated/obsolete settings
* Removes invalid reference so that we won't break the docs builds